trace/trace.go

```diff
@@ -48,7 +48,7 @@ func Setup(name, ver, agent, typ string, prm float64, buckets []float64) error {
 			LocalAgentHostPort:  agent,
 		},
 	}
-	time.Sleep(100 * time.Millisecond)
```

Comment:
I thought this is unnecessary, but I can put it back since it is not strictly related to what I am implementing here.

Reply:
It was added for a reason i assume which I do not remember TBH.